### PR TITLE
First Send Screen

### PR DESCRIPTION
**Pull request by 0xLe0x**

### CODE REVIEW

File: packages/app/features/send/screen.tsx [hunk 1]
```diff
@@ -4,54 +4,107 @@ import {
...
-            </H1>
-            <XStack pos={'absolute'} r={0} space={'$1.5'}>
-              <Paragraph theme={'alt2'}>Bal</Paragraph>
-              <Paragraph fontWeight={'700'}>1.25</Paragraph>
+    <>
+      <MainLayout>
+        <Container>
+          <YStack maw={316} pt={'$13'}>
+            <XStack jc={'center'}>
+              <H1 size={sendAmount.length > 4 ? sendAmount.length > 8 ? '$9' : '$12' : '$15'}>
```
Comment (0xBigBoss) [inline packages/app/features/send/screen.tsx:65]: i like the dynamic size, but it causes the UI to shift. I think give it's container the max height possible so we avoid the layout shift.

**Review by 0xBigBoss — COMMENTED** [review]
sry one more thing i forgot

File: packages/app/features/send/components/send-modal/send-it.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    <Theme name={resolvedTheme as ThemeName}>
+      <Dialog
+        modal
+        onOpenChange={(open) => {
+          setCurrentModal('')
+        }}
+        open={showModal}
+      >
+        <Adapt when="sm" platform="touch">
+          <Sheet animation="medium" zIndex={200000} modal dismissOnSnapToBottom snapPoints={[100]}>
```
[the remark this answers is not in the document]
Comment (0xLe0x) [reply packages/app/features/send/components/send-modal/send-it.tsx:45]: Totally agree

File: packages/app/features/send/screen.tsx (hunk 1 above)
Comment (0xLe0x) [reply packages/app/features/send/screen.tsx:65]: 👍

File: packages/app/features/send/components/numpad/index.tsx
[the remark this answers is not in the document]
Comment (0xLe0x) [reply packages/app/features/send/components/numpad/index.tsx:1]: fixed

File: packages/ui/src/config/media.ts [hunk 3]
```diff
@@ -13,6 +13,10 @@ export const media = createMedia({
   gtMd: { minWidth: 1020 + 1 },
   gtLg: { minWidth: 1280 + 1 },
   gtXl: { minWidth: 1650 + 1 },
+
+  shorter: { maxHeight: 820 },
```
Comment (0xBigBoss) [inline packages/ui/src/config/media.ts:17]: ty 🤩

**Review by 0xBigBoss — APPROVED** [review]
lgtm

**Pull request merged**